.devops/messages-code-review-pipelines.yml

```diff
@@ -0,0 +1,139 @@
+pr:
+  autoCancel: false
+  branches:
+    include:
+      - main
+  paths:
+    include:
+      - 'src/domains/idpay-app'
+      - 'src/domains/idpay-common'
```

Comment:
```suggestion
      - 'src/domains/idpay-app'
      - 'src/domains/idpay-common'
```
typo idpay

Reply:
right now this file is only a place holder